### PR TITLE
Add `supportedFeatures` field to build remotes

### PR DESCRIPTION
**Pull request by ryantrinkle**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -154,6 +162,7 @@
         hostName = "localhost";
         systems = config.simple-hydra.localBuilder.systems;
         maxJobs = config.simple-hydra.localBuilder.maxJobs;
+	supportedFeatures = config.simple-hydra.localBuilder.supportedFeatures;
```
Comment (ElvishJerricco) [inline default.nix]: Indentation ;)

**Comment by ElvishJerricco** [comment]
Ooh cool, thanks!

File: default.nix (hunk 1 above)
Comment (ryantrinkle) [reply default.nix]: Oh, weird!  That's what i get for pushing from a machine with no emacs config :P

**Comment by ryantrinkle** [comment]
Fixed the indentation (replaced hard tab with spaces)

**Pull request merged**